main.go

```diff
@@ -32,16 +34,15 @@ func main() {
 	h.SetIsRemote(true)
 	h.SetDeviceSerialNumber(597101)
 	h.SetHubPort(0)
-	err = h.OpenWaitForAttachment(2000)
-	if err != nil {
+	if err := h.OpenWaitForAttachment(time.Second * 2); err != nil {
 		panic(err)
 	}
-	h.SetOnHumidityChangeHandler(func(p phidgets.Phidget, ctx interface{}, value float32) {
+	h.SetOnHumidityChangeHandler(func(value float64) {
```

Comment:
Good point on removing the context concept for Go, since go has closures and C does not, having ctx just adds confusion.